src/js/helpers/local_storage.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+var default_cookies = require('./cookies');
+var local_storage_session_length = 0;
+
+module.exports = Object.assign({}, default_cookies, {
+
+  set: function(name, value, minutes, domain, excl_subdomains) {
+    // Don't break the build
+    domain = '';
+    excl_subdomains = false;
+
+    // `item` is an object which contains the original value
+    // as well as the time when it's supposed to expire
+    var item = {
+      value: this.encodeData( value ),
+      expiry: (new Date()).getTime() + ( Math.max( minutes, local_storage_session_length ) * 60 * 1000 ),
+    };
+    localStorage.setItem(name, JSON.stringify(item));
+  },
+
+  get: function(name) {
+    var item_raw = localStorage.getItem(name);
+    // if the item doesn't exist, return null
+    if (!item_raw) {
```

Comment:
Actually, the code would do "if the item is falsy, return null", and `localStorage.getItem` returns `null` for non-existing items.

Is that the case that we consider [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) values (`", 0, false, ...`) "non-existing"?

If so, I'd rather rephrase the code comment to explain from the "why?" perspective, not the "what?" perspective. ("What?" is answered in the code.)


```suggestion
    // We consider items set to a falsy value as nonexistent and return `null` consistently.
    if (!item_raw) {
```

But, I don't know if that's the actual intention.

Reply:
Or if we want to just forward the behavior, and return null for non existing, I'd rather change the code:
```suggestion
    // if the item doesn't exist, return null
    if (!item_raw === null) {
```

Currently, we do set the item to be stringified JSON object, so it will always be nonfalsy if existing, but I'd rather make the code resilient for future changes. Plus, somebody may use this method to ask for an object set by a different piece of software.